scale/job/migrations/0034_auto_20180112_1759.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.1 on 2018-01-12 17:59
+from __future__ import unicode_literals
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('job', '0033_auto_20180105_2113'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='job',
+            old_name='cpus_required',
+            new_name='input_file_size',
+        ),
+        migrations.RemoveField(
+            model_name='job',
+            name='data',
+        ),
```

Comment:
It should be `disk_in_required` being renamed to `input_file_size`.
Also `data` and `results` need to be renamed to `input` and `output`. This migration will drop these columns losing all job input and output information.